pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/StandardTextValues.java

```diff
@@ -30,6 +30,7 @@
     CONFIGURATION_TEST("test"),
     CONFIGURATION_VENV("venv"),
     CONFIGURATION_WHEEL("wheel"),
+    CONFIGURATION_FLAKE8("flake8"),
```

Comment:
This name will need to be changed if we rename this configuration as I suggested.